tests/rundoctest.py

```diff
@@ -1,17 +1,23 @@
 #!/usr/bin/env python
 
+import os
 import sys
 import cypari2
 import doctest
 
+path = os.path.dirname(__file__)
+if path:
+    os.chdir(path)
+
 failed = 0
 attempted = 0
 for mod in [cypari2.closure, cypari2.convert, cypari2.gen,
-            cypari2.handle_error, cypari2.pari_instance, cypari2.stack]:
+            cypari2.handle_error, cypari2.pari_instance, cypari2.stack,
+            cypari2.string_utils]:
 
     print("="*80)
     print("Testing {}".format(mod.__name__))
-    test = doctest.testmod(mod, optionflags=doctest.ELLIPSIS|doctest.REPORT_NDIFF)
+    test = doctest.testmod(mod, optionflags=doctest.ELLIPSIS|doctest.REPORT_NDIFF|doctest.IGNORE_EXCEPTION_DETAIL)
```

Comment:
It is. In Python3, pari error are printed

    Traceback (most recent call last):
    ...
    cypari2.handle_error.PariError: MSG

while in Python2 it is

    Traceback (most recent call last):
    ...
    PariError: MSG